docs/source/services/timelock_service/installation.rst

```diff
@@ -50,3 +71,52 @@ Timelock Server Installation
               "healthy": true
           }
       }
+
+Add client(s) to Timelock
+-------------------------
+
+8. Back up each client, following the steps declared in :ref:`backup-restore`.
+
+9. Configure each client to use Timelock.
+   Detailed documentation is :ref:`here <timelock-client-configuration>`.
+   You must remove any ``leader``, ``timestamp``, or ``lock`` blocks; the ``timelock`` block to add looks like this:
+
+.. code-block:: yaml
+
+   atlasdb:
+      timelock:
+        client: tom
+        serversList:
+          servers:
+            - palantir-1.com:8080
+            - palantir-2.com:8080
+            - palantir-3.com:8080
+          sslConfiguration:
+            trustStorePath: var/security/truststore.jks
+
+10. Configure Timelock to respond to your clients.
+    Add each client as a named entry in the ``clients`` block of your :ref:`timelock-server-configuration`.
+
+
+.. code-block:: yaml
+
+   clients:
+      - tom
+      - jerry
+
+10. As a verification step, request a fresh timestamp from each client. We will use these values later to check that the migration is complete.
+
+11. Shut down each client that has been newly added.
+
+12. Restart your Timelock cluster.
+
+13. Migrate each client to the timelock server - see the :ref:`separate migration docs <timelock-migration>`. For Cassandra KVS, this is automatic.
+
+.. warning::
+
+    Do not skip this step if your client uses DbKvs! Failure to migrate your client will cause **severe data corruption**, as Timelock will serve timestamps starting from 1.
+
+14. Restart each client.
+
+15. To verify that the migration worked correctly, get a fresh timestamp for each client from the Timelock server.
+    For each client, the timestamp returned should be strictly greater than the timestamps obtained in step 10.
```

Comment:
nit: Is probably obvious, but to avoid confusion I'd suggest being even more explicit here (i.e. "timestamp returned should be strictly greater than the _corresponding timestamp_ obtained in step 10") - don't want someone to panic because they had {Tom: 10, Jerry: 50} and now see 11 for Tom expecting it to be at least 51.